Makefile

```diff
@@ -1,75 +1,27 @@
-all: init test build
-build: init docs/index.yaml docs/es/all.yaml docs/es/all-latest.yaml
+# Top-level Makefile for working on all helm-charts projects at once.
 
-CHART := enterprise-suite
-export VERSION := $(shell scripts/export-chart-version.sh $(CHART))
-export RELEASE := $(CHART)-$(VERSION)
-CHART_LATEST := $(CHART)-latest
-export RELEASE_LATEST := $(CHART_LATEST)-$(VERSION)
+# Collection of charts to process.
+# These are subdirectories of helm-charts.  Add to the list as required.
+#
+# This setting is the default.  To build a subset (at this level), just define a
+# value on the command line.  e.g. "make CHARTS=sample-project"
+#
+CHARTS = enterprise-suite enterprise-suite-latest reactive-sandbox #sample-project
 
-define banner
-	$(info === $@)
-endef
 
-docs/es/all.yaml: docs/$(RELEASE).tgz
-	$(call banner)
-	helm --namespace=lightbend template $< > $@
+TOPTARGETS := all build clean test lint-helm minikube-test
```

Comment:
lint-helm should be in the ES Makefile. Just generic targets only in this top level Makefile.

Reply:
The whole point of this exercise is to allow users to add helm-charts to some repository.  How is `lint-helm` ES-specific?  That would apply to all clients no?  (I did make `lint-json` and `lint-promql` ES-specific though.)

Reply:
Thought about this a bit more...  I think the `lint-helm` target makes sense in `common.mk` so that all clients can use it.  I realize now that your comment is on the occurrence in the top-level `Makefile` though.  I think it's reasonable there, so one could easily run it across all the projects at once, but I could be convinced to remove it from the top-level.

Reply:
We don't have any say in how other teams test their helm charts. We shouldn't be making anything common unless someone else wants it first. Premature optimization and all that.